lib/migration.ts

```diff
@@ -0,0 +1,15 @@
+import { readdir } from "node:fs/promises";
+import { join } from "node:path";
+import { FileMigrationProvider, Migrator } from "kysely";
+import { db } from "./db";
+
+const migrator = new Migrator({
+	db,
+	provider: new FileMigrationProvider({
+		fs: { readdir: (path: string) => readdir(path, "utf-8") },
```

Comment:
I commented on this in the [project already](https://github.com/users/indifferentghost/projects/7/views/1?pane=issue&itemId=47620771) I would prefer to go _all in_ on bun.

Reply:
Created https://github.com/indifferentghost/tamagetajob/issues/6